Common/src/Common/RefData.php

```diff
@@ -212,6 +212,7 @@ class RefData
     public const PERMISSION_CAN_MANAGE_USER_INTERNAL = 'can-manage-user-internal';
     public const PERMISSION_SELFSERVE_EBSR_UPLOAD = 'selfserve-ebsr-upload';
     public const PERMISSION_SELFSERVE_EBSR_DOCUMENTS = 'selfserve-ebsr-documents';
+    public const PERMISSIONS_CAN_LIST_CONVERSATIONS = 'can-list-conversations';
 
```

Comment:
This is a good idea, however can we change to the following:
```php
    public const PERMISSION_CAN_LIST_CONVERSATIONS = 'can-list-conversations';
    public const PERMISSION_CAN_LIST_MESSAGES = 'can-list-messages';
    public const PERMISSION_CAN_REPLY_TO_CONVERSATIONS = 'can-reply-to-conversation';
    public const PERMISSION_CAN_CREATE_CONVERSATION = 'can-create-conversation';
    public const PERMISSION_CAN_DISABLE_MESSAGING = 'can-disable-messaging';
    public const PERMISSION_CAN_ENABLE_MESSAGING = 'can-enable-messaging';
    public const PERMISSION_CAN_CLOSE_CONVERSATION = 'can-close-conversation';
```

(because you've used the plural, and it will help solve a related issue in selfserve)

Reply:
What do you mean?

Reply:
One niggle...
Change:
```php
public const PERMISSION_CAN_REPLY_TO_CONVERSATIONS = 'can-reply-to-conversation';
```
to
```php
public const PERMISSION_CAN_REPLY_TO_CONVERSATION = 'can-reply-to-conversation';
```
(ought to be singular)

Reply:
Gramatically it makes more sense with the S, a lot of permissions are in the plural form so I think the singular/plural use isn't a big deal.

Reply:
Primarily, we just want the const to match the string.

However, what if we had a change that allowed a user to reply to many conversations at once?
Then we'd need a separate permission, but if we use the plural now, the name would already have been used.

This is where grammar rules get overidden, because we want to be very specific about multiple entities vs singular ones that are acted on.